pkg/rtc/participant.go

```diff
@@ -1618,12 +1639,15 @@ func (p *ParticipantImpl) DebugInfo() map[string]interface{} {
 }
 
 func (p *ParticipantImpl) GetDataTrack() types.DataTrack {
+	p.lock.RLock()
+	defer p.lock.RUnlock()
+
 	return p.dataTrack
```

Comment:
I am guessing we might need the same interface check here?

Reply:
yeah, need same check here

Reply:
made the change in this commit - https://github.com/livekit/livekit-server/pull/508/commits/ef1cdcf58eab0055ba4c17553c838624eb96f2cf